modules/nf-core/wittyer/main.nf

```diff
@@ -0,0 +1,70 @@
+process WITTYER {
+    tag "$meta.id"
+    label 'process_single'
+
+    container "nf-core/wittyer:0.3.3.0"
+
+    // Exit if running this module with -profile conda / -profile mamba
+    if (workflow.profile.tokenize(',').intersect(['conda', 'mamba']).size() >= 1) {
+        error "WITTYER module does not support Conda. Please use Docker / Singularity / Podman instead."
+    }
+
+    input:
+    // use stageAs to allow comparing the same vcfs against each other
+    tuple val(meta), path(query_vcf, stageAs: "query.vcf.gz"), path(query_vcf_index, stageAs: "query.vcf.gz.tbi"), path(truth_vcf, stageAs: "truth.vcf.gz"), path(truth_vcf_index, stageAs: "truth.vcf.gz.tbi"), path(bed)
```

Comment:
Don't use `stageAs` here, it's easier to debug modules if they retain the original filenames :)

Reply:
I wanted to test truth and query with the same files. Because I find it easier to check if my benchmarking workflow works when I can compare a file against itself. So if I remove it here it would need to give them different names?

Reply:
Yeah but at the cost of making it harder on yourself when you want to actually use the process? Can't you make a copy of the query file and rename it before running the pipeline? :p

Reply:
I'll change it  and adapt the test accordingly